nylas/client/client.py

```diff
@@ -62,6 +63,8 @@ def __init__(self, app_id=environ.get('NYLAS_APP_ID'),
         self.app_secret = app_secret
         self.app_id = app_id
 
+        self.scopes = scopes or ['email']
```

Comment:
Don't we want to default to all scopes?
```suggestion
        self.scopes = scopes or ['email', 'calendar', 'contacts]
```

Reply:
Depends on which we care about more: backwards compatibility, or consistency with the other SDKs. Right now, the SDK is sending the `scope` (singular) parameter set to "email", so the most backwards-compatible option is to continue sending that by default.

However, I'm inclined to agree that switching the default to all scopes is probably a better change. It's probably what our users will expect, and it's consistent with the other SDKs as well.